docs/get-started/deployment-and-usage.md

```diff
@@ -148,6 +156,13 @@ The command deploys Node Feature Discovery on the Kubernetes cluster in the
 default configuration.  The Configuration section describes how it can be
 configured during installation.
 
+In order to deploy the [minimal](#minimal) image you need to override the image
+tag:
+
+```bash
+helm install node-feature-discovery ./node-feature-discovery/ --set image.tag={{ site.release }}-minimal --namespace $NFD_NS --create-namespace
```

Comment:
It's not a valid bash command. Where {{site.release}} is  defined?

Reply:
It comes from the Jekyll config:
https://github.com/kubernetes-sigs/node-feature-discovery/blob/ea54b7b98a758b9629471bd4abb8294f9943a04f/docs/_config.yml#L49

Reply:
Thanks for the explanation